### PR TITLE
added: save result points in AdaptiveSIM

### PR DESCRIPTION
**Pull request by akva2**

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this ifem-navierstokes=111 please

**Comment by akva2** [comment]
jenkins build this with downstreams ifem-navierstokes=111 please

File: src/SIM/AdaptiveSIM.C [hunk 1]
```diff
@@ -212,6 +212,7 @@ bool AdaptiveSIM::adaptMesh (int iStep, std::streamsize outPrec)
 
 bool AdaptiveSIM::writeGlv (const char* infile, int iStep)
 {
+  this->savePoints(0.0, iStep);
```
Comment (kmokstad) [inline src/SIM/AdaptiveSIM.C:215]: The writeGlv method should strictly deal with VTF-output only. Is there somewhere else you can put that savePoints call?

Comment (akva2) [reply src/SIM/AdaptiveSIM.C]: sure, I put it in the solveStep method instead.

**Comment by akva2** [comment]
jenkins build this with downstreams ifem-navierstokes=111 please

**Comment by akva2** [comment]
jenkins build this with downstreams ifem-navierstokes=111 please

**Pull request merged**